docs/contributor_guide/testing.md

```diff
@@ -0,0 +1,22 @@
+# Testing
+
+## Python unit tests
+
+Requires `pytest` and `dirty-equals`:
+
+```bash
+pip install pytest dirty-equals
+```
+
+Run the tests:
+
+```bash
+pytest --color=yes -v python
+```
+
+## Integration tests
+
+We don't recommend running integration tests locally because they are browser-based
```

Comment:
I think it's better to run ui-tests locally when you're working on adding/modifying a new or existing ui test essentially to avoid the time CI takes to run the tests.

Reply:
:heavy_check_mark: Will update!

Reply:
8fac61d